index.rst

```diff
@@ -0,0 +1,267 @@
+..  Content of technical report.
+
+  See http://docs.lsst.codes/en/latest/development/docs/rst_styleguide.html
+  for a guide to reStructuredText writing.
+
+  Do not put the title, authors or other metadata in this document;
+  those are automatically added.
+
+  Use the following syntax for sections:
+
+  Sections
+  ========
+
+  and
+
+  Subsections
+  -----------
+
+  and
+
+  Subsubsections
+  ^^^^^^^^^^^^^^
+
+  To add images, add the image file (png, svg or jpeg preferred) to the
+  _static/ directory. The reST syntax for adding the image is
+
+  .. figure:: /_static/filename.ext
+     :name: fig-label
+     :target: http://target.link/url
+
+     Caption text.
+
+   Run: ``make html`` and ``open _build/html/index.html`` to preview your work.
+   See the README at https://github.com/lsst-sqre/lsst-report-bootstrap or
+   this repo's README for more info.
+
+   Feel free to delete this instructional comment.
+
+:tocdepth: 1
+
+This document describes a framework which was created to test how well shape measurement algorithms in the DM Stack can measure galaxy shear.  The targets algorithms for this framework are in the package meas_modelfit. This package houses both the CModel algorithm -- used to measure galaxy shapes) and the ShapeletPsfApprox Algorithm (SPA) -- used to create a parameterized model of the Psf (point spread function). However, the framework is set up to allow similar tests on other shape measurment algorithms.
+
+Shear measurement was done on images created by Great3Sims and GalSim as described below.  However, the Point Spread Function (Psf) which was applied to the simulated galaxies was created by PhoSim using an LSST simulation.
+
+Our goal was to run both Psf approximation and shape measurement algorithms on large numbers of galaxies that have been subjected to known constant shear and Psf distortions, and to characterize the shear bias of these algorithms.
```

Comment:
Better to say "characterize the changes in shear bias of these algorithms due to different approximations".